### PR TITLE
docs: Modify the doc following the v0.5 release, part 1

### LINKED ISSUE #1007: docs: Modify the doc following the v0.5 release, part 1
Taking into account comments from @tuscland, @MarieS-WiMLDS and @sylvaincom (see #1008 for the second iteration):

- README
    - [x] update the cross-validation example
- overall
    - [x] the logo on the top left is too big
    - [x] change Twitter into X
    - [x] add Bluesky
    - [x] add LinkedIn
    - [x] add YouTube
    - [x] "Probabl website" -> "Probabl"
- landing page
    - [x] the logo is too small
    - [x] remove the parts about starring
    - [x] remove the badges
    - [x] the orange color is not the correct one
    - [x] remove the right side bar
- install
    - [x] change the tab appearance
    - [x] "Skore" with uppercase if at the beginning of a sentence
    - [x] remove the what's next
- user guide
    - [x] "examples" -> "user guide"
    - [x] "Quick start": to be created
    - [x] "Getting started with skore" -> "Skore product tour"
    - [x] "Tracking items using their history" -> "Tracking items"
    - [x] "Overview of the skore project" -> "Working with projects"
    - [x] "Enhancing cross-validation" -> "Cross-validation"
    - [x] "Enhancing train-test split" -> "Train-test split""
    - [x] remove the "Track" section
    - [x] in the tracking example, rename `item_history` into `passed_item `
    - [x] rename "Diagnose" into "Model evaluation" 
- contributing
    - [x] Change the first sentence into "Thank you for your interest in contributing to Skore! We welcome contributions from everyone and appreciate you taking the time to get involved."

### PR DESCRIPTION
**Pull request by sylvaincom**
Fix #1007

### CODE REVIEW

**Comment by glemaitre** [comment]
@sylvaincom side note: you should more context in the summary without the need to go in the original issue.

File: examples/getting_started/plot_quick_start.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+# %%
+# Evaluate your model using skore's :class:`~skore.CrossValidationReporter`:
+
+# %%
+from sklearn.datasets import load_iris
+from sklearn.svm import SVC
+
+X, y = load_iris(return_X_y=True)
+clf = SVC(kernel="linear", C=1, random_state=0)
```
Comment (glemaitre) [inline examples/getting_started/plot_quick_start.py:29]: Do not use an SVC but rather a LogisticRegression

File: examples/README.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .. _auto_examples:
 
-Examples
-========
+User guide
+==========
 
 Below is a gallery of narrated notebook examples on how and why to use skore.
-They also serve as our user guide.
+They serve as our user guide.
```
Comment (glemaitre) [inline examples/README.txt:7]: I have a really strong opinion on this: by not forcing ourself to write the user guide, we are just going to drop a couple of line of example and state that this is good enough.

We should really write a user guide to give the perspective on our tools.

File: examples/getting_started/plot_quick_start.py (hunk 1 above)
Comment (sylvaincom) [reply examples/getting_started/plot_quick_start.py:29]: With the following code:
```python
from sklearn.datasets import load_iris
from sklearn.linear_model import LogisticRegression

X, y = load_iris(return_X_y=True)

clf = LogisticRegression()

reporter = skore.CrossValidationReporter(clf, X, y, cv=5)
```

I get the following error:
```
/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/model_selection/_validation.py:1000: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details:
Traceback (most recent call last):
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 139, in __call__
    score = scorer._score(
            ^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 376, in _score
    return self._sign * self._score_func(y_true, y_pred, **scoring_kwargs)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/utils/_param_validation.py", line 213, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_classification.py", line 2968, in log_loss
    raise ValueError(
ValueError: y_true and y_pred contain different number of classes 3, 2. Please provide the true labels explicitly through the labels argument. Classes found in y_true: [0 1 2]

  warnings.warn(
/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/linear_model/_logistic.py:469: ConvergenceWarning: lbfgs failed to converge (status=1):
STOP: TOTAL NO. of ITERATIONS REACHED LIMIT.

Increase the number of iterations (max_iter) or scale the data as shown in:
    https://scikit-learn.org/stable/modules/preprocessing.html
Please also refer to the documentation for alternative solver options:
    https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression
  n_iter_i = _check_optimize_result(
/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/model_selection/_validation.py:1000: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details:
Traceback (most recent call last):
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 139, in __call__
    score = scorer._score(
            ^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 376, in _score
    return self._sign * self._score_func(y_true, y_pred, **scoring_kwargs)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/utils/_param_validation.py", line 213, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_classification.py", line 2968, in log_loss
    raise ValueError(
ValueError: y_true and y_pred contain different number of classes 3, 2. Please provide the true labels explicitly through the labels argument. Classes found in y_true: [0 1 2]

  warnings.warn(
/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/model_selection/_validation.py:1000: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details:
Traceback (most recent call last):
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 139, in __call__
    score = scorer._score(
            ^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 376, in _score
    return self._sign * self._score_func(y_true, y_pred, **scoring_kwargs)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/utils/_param_validation.py", line 213, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_classification.py", line 2968, in log_loss
    raise ValueError(
ValueError: y_true and y_pred contain different number of classes 3, 2. Please provide the true labels explicitly through the labels argument. Classes found in y_true: [0 1 2]

  warnings.warn(
/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/model_selection/_validation.py:1000: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details:
Traceback (most recent call last):
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 139, in __call__
    score = scorer._score(
            ^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 376, in _score
    return self._sign * self._score_func(y_true, y_pred, **scoring_kwargs)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/utils/_param_validation.py", line 213, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_classification.py", line 2968, in log_loss
    raise ValueError(
ValueError: y_true and y_pred contain different number of classes 3, 2. Please provide the true labels explicitly through the labels argument. Classes found in y_true: [0 1 2]

  warnings.warn(
/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/model_selection/_validation.py:1000: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details:
Traceback (most recent call last):
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 139, in __call__
    score = scorer._score(
            ^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_scorer.py", line 376, in _score
    return self._sign * self._score_func(y_true, y_pred, **scoring_kwargs)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/utils/_param_validation.py", line 213, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sylvaincom/Documents/GitHub/skore/.venv/lib/python3.12/site-packages/sklearn/metrics/_classification.py", line 2968, in log_loss
    raise ValueError(
ValueError: y_true and y_pred contain different number of classes 3, 2. Please provide the true labels explicitly through the labels argument. Classes found in y_true: [0 1 2]

  warnings.warn(
```

File: examples/README.txt (hunk 2 above)
Comment (sylvaincom) [reply examples/README.txt:7]: Yes, we are thinking of doing a user guide in addition to an examples tab, but do we currently have enough features for a user guide?

File: examples/getting_started/plot_quick_start.py (hunk 1 above)
Comment (glemaitre) [reply examples/getting_started/plot_quick_start.py:29]: Yes it is bug introduce some times ago. I open a PR to fix it here: https://github.com/probabl-ai/skore/pull/1009

File: examples/README.txt (hunk 2 above)
Comment (sylvaincom) [reply examples/README.txt:7]: Created a separate issue: https://github.com/probabl-ai/skore/issues/1017

File: examples/getting_started/plot_quick_start.py (hunk 1 above)
Comment (sylvaincom) [reply examples/getting_started/plot_quick_start.py:29]: Created a separate issue: https://github.com/probabl-ai/skore/issues/1018

**Comment by sylvaincom** [comment]
@glemaitre thanks your for comments, do you think we could merge this soon? And I'll get back to the separate issues when I have time? (I'm off today and tomorrow)

cc @MarieS-WiMLDS

**Comment by glemaitre** [comment]
> thanks your for comments, do you think we could merge this soon?

I have no objection. We can have subsequent PR to further improve (the documentation is never perfect ;) )

**Review by MarieS-WiMLDS — APPROVED** [review]

**Pull request merged**